src/query/catalog/src/database.rs

```diff
@@ -41,4 +63,115 @@ pub trait Database: DynClone + Sync + Send {
     async fn init_database(&self, _tenant: &str) -> Result<()> {
         Ok(())
     }
+
+    // Build a `Arc<dyn Table>` from `TableInfo`.
+    fn get_table_by_info(&self, _table_info: &TableInfo) -> Result<Arc<dyn Table>> {
```

Comment:
```suggestion
    fn get_table_by_info(&self, table_info: &TableInfo) -> Result<Arc<dyn Table>> {
    let _ = table_info;
```

For better reading in IDE.